modules/terraform/Makefile

```
@@ -125,5 +131,5 @@ terraform/v14-rewrite: terraform/loosen-constraints terraform/bump-tf-12-min-ver
 	@{ [[ "$(TERRAFORM_FORCE_README)" != "true" ]] && git diff --no-patch --exit-code README.yaml; } || $(MAKE) readme
 
 ## Terraform pull-request routine check/update
-terraform/precommit: terraform/fmt terraform/tflint readme/build
+terraform/precommit: terraform/tflint-fix terraform/fmt terraform/tflint readme/build
```

Let's not run tflint fixes by default.

```suggestion
terraform/precommit: terraform/fmt terraform/tflint readme/build
```